lektor/imagetools.py

```diff
@@ -344,7 +349,16 @@ def get_quality(source_filename):
     return 85
 
 
-def computed_height(source_image, width, actual_width, actual_height):
+def computed_height(source_image, format, width, actual_width, actual_height):
+    # If the file is a JPEG file and the EXIF header shows that the image
+    # is rotated, imagemagick will auto-orient the file. That is, a 400x200
+    # file where the target width is 100px will *not* be converted to a
+    # 100x50 image, but to 100x200.
+    if format == 'jpeg':
+        exif = read_exif(open(source_image, 'rb'))
+        if exif.rotated:
+            return int(float(actual_width) *
```

Comment:
Instead of re-implementing the logic on this line that _also_ exists on the next line, can you just swap the values?
```python
if exif.rotated:
    actual_width, actual_height = actual_height, actual_width
return int(float(actual_height) * (float(width) / float(actual_width)))
```